projects/canopy/src/lib/spacing/padding/padding.directive.ts

```diff
@@ -14,69 +14,78 @@ import { SpacingVariant } from '../spacing.interface';
   `,
 })
 export class LgPaddingDirective {
-  paddingTopClass: string;
+  paddingTopClasses: Array<string> = [];
   @Input()
-  set lgPaddingTop(padding: SpacingVariant) {
-    this.paddingTopClass = this.togglePaddingClass(
-      `lg-padding__top--${padding}`,
-      this.paddingTopClass,
-    );
+  set lgPaddingTop(padding: SpacingVariant | ResponsiveSpacing) {
+    const newClasses = this.createNewClasses(padding, 'lg-padding__top');
+    this.paddingTopClasses = this.toggleClasses(newClasses, this.paddingTopClasses);
   }
 
-  paddingRightClass: string;
+  paddingRightClasses: Array<string> = [];
   @Input()
-  set lgPaddingRight(padding: SpacingVariant) {
-    this.paddingRightClass = this.togglePaddingClass(
-      `lg-padding__right--${padding}`,
-      this.paddingRightClass,
-    );
+  set lgPaddingRight(padding: SpacingVariant | ResponsiveSpacing) {
+    const newClasses = this.createNewClasses(padding, 'lg-padding__right');
```

Comment:
Minor stylistic suggestion, but might not be to everyones tastes :-)
```suggestion
    this.paddingRightClasses = this.toggleClasses(
      this.createNewClasses(padding, 'lg-padding__right'),
      this.paddingRightClasses
    );
```

Reply:
Done ✅